tests/unit/operators/test_datarobot.py

```diff
@@ -131,3 +137,55 @@ def test_operator_score_predictions(mocker):
 
     assert job_id == "job-id"
     score_mock.assert_called_with("deployment-id", **settings)
+
+
+@pytest.fixture
+def drift_details():
+    return {
+        "period": {
+            "start": datetime.fromisoformat("2000-01-01"),
+            "end": datetime.fromisoformat("2000-01-07"),
+        },
+        "drift_score": 0.9,
+    }
+
+
+def test_operator_get_target_drift(mocker, drift_details):
+    deployment_id = "deployment-id"
+    from datarobot.models.data_drift import TargetDrift
+
+    target_drift = TargetDrift(**drift_details)
+    expected_target_drift = _serialize_drift(TargetDrift(**drift_details))
+
+    mocker.patch.object(dr.Deployment, "get", return_value=dr.Deployment(deployment_id))
+    mocker.patch.object(TargetDrift, "get", return_value=target_drift)
+
+    operator = GetTargetDriftOperator(
+        task_id="score_predictions", deployment_id=deployment_id
+    )
+
+    drift = operator.execute(context=dict())
+
+    assert type(drift) == dict
+    assert drift == expected_target_drift
+
+
+def test_operator_get_feature_drift(mocker, drift_details):
+    deployment_id = "deployment-id"
+    from datarobot.models.data_drift import FeatureDrift
+
+    feature_drift = FeatureDrift(**drift_details)
+    expected_feature_drift = [_serialize_drift(FeatureDrift(**drift_details))]
```

Comment:
I always suggest testing with at least two items in the list.

Reply:
@zingbretsen Just want to stress here how important is to always test with multiple list items. I've added another item and the test failed, as there was a bug in `_serialize_drift()` where it was changing the attributes of the original objects which resulted in errors later on, as the code expected start and date to be datetime. The fix is:

```
drift_dict = drift_obj.__dict__.copy()
```